CHANGELOG.md

```diff
@@ -1,9 +1,21 @@
 # Changelog
 
 tbd
-- New Class `SignatureRequestFrom` to handle signature requests by reference
-- Rename `AuthenticationRequestParser` to `RequestParser`
-    - `RequestParser` can now handle `SignatureRequestFrom`
+- New `Initializer` object in `vck-openid` which needs to be called at the start of the project if artifact is used
```

Comment:
Is this also needed when using "plain" OpenID data classes only?

Reply:
In case we only want to use the OpenId data class module we can use the `odcJsonSerializer` found therein. The initializer adds the serialization modules of that to `vckJsonSerializer` which only makes sense if we are aware of both the OpenId data module and the vck-main module at compile time.